docs/architecture/cloud-native/infrastructure-as-code.md

```diff
@@ -85,14 +87,38 @@ resource "azurerm_storage_account" "testsa" {
 }
 ```
 
-**Figure 11-12** - An example of a Resource Manager template
+**Figure 10-16** - An example of a Resource Manager template
+
+Terraform also provides more sensible error messages for problem templates. There's even a handy validate task that can be used in the build phase to catch template errors early. Errors with ARM can be challenging to understand.
```

Comment:
Errors with ARM can be challenging - If a reader has never tried this before, it is hard to understand. I think we should rephrase this line and stress on what Terraform provides.

Reply:
Fixed

Reply:
Fixed